_layouts/default.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="{{ site.lang | default: "en-US" }}">
+  <head>
+    <meta charset="UTF-8">
+
+{% seo %}
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preload" href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" as="style" type="text/css" crossorigin>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#157878">
+    <link rel="icon" href="/_includes/favicon.png">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    {% include head-custom.html %}
+  </head>
+  <body>
+    <a id="skip-to-content" href="#content">Skip to the content.</a>
```

Comment:
<img width="175" alt="Screenshot 2023-04-13 at 15 46 35" src="https://user-images.githubusercontent.com/97447607/231797199-3ca049bd-21f6-45e6-b949-b9e298042aa5.png">
Is this link the button in the image?

Reply:
Ah nvm realise this whole thing is the header basically.

Reply:
Yeah, I've hardcoded the button and the logo in the header, the rest of it comes from the `_config.yml` file or is just a core part of the theme.